akka-cluster/src/multi-jvm/scala/akka/cluster/DowningWhenOtherHasQuarantinedThisActorSystemSpec.scala

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (C) 2009-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster
+
+import akka.actor.ActorRef
+import akka.actor.Identify
+import akka.actor.RootActorPath
+
+import scala.concurrent.duration._
+import akka.remote.artery.ArterySettings
+import akka.remote.artery.ThisActorSystemQuarantinedEvent
+import akka.remote.testkit.MultiNodeConfig
+import akka.remote.testkit.MultiNodeSpec
+import akka.remote.transport.ThrottlerTransportAdapter
+import akka.testkit.LongRunningTest
+import com.typesafe.config.ConfigFactory
+
+object DowningWhenOtherHasQuarantinedThisActorSystemSpec extends MultiNodeConfig {
+  val first = role("first")
+  val second = role("second")
+  val third = role("third")
+
+  commonConfig(
+    debugConfig(on = false)
+      .withFallback(MultiNodeClusterSpec.clusterConfig)
+      .withFallback(
+        ConfigFactory.parseString("""
+        akka.remote.artery.enabled = on
+        akka.cluster.downing-provider-class = "akka.cluster.sbr.SplitBrainResolverProvider"
+        # speed up decision
+        akka.cluster.split-brain-resolver.stable-after = 5s
+        """)))
+
+  // exaggerate the timing issue by ,making the second node decide slower
+  // this is to more consistently repeat the scenario where the other side completes downing
+  // while the isolated part still has not made a decision and then see quarantined connections from the other nodes
+  nodeConfig(second)(ConfigFactory.parseString("akka.cluster.split-brain-resolver.stable-after = 15s"))
+
+  testTransport(on = true)
+}
+
+class DowningWhenOtherHasQuarantinedThisActorSystemMultiJvmNode1 extends SplitBrainQuarantineSpec
+class DowningWhenOtherHasQuarantinedThisActorSystemMultiJvmNode2 extends SplitBrainQuarantineSpec
+class DowningWhenOtherHasQuarantinedThisActorSystemMultiJvmNode3 extends SplitBrainQuarantineSpec
```

Comment:
wrong extends, should be extends DowningWhenOtherHasQuarantinedThisActorSystemSpec

Reply:
Ah, yes, I fixed that friday but never pushed.

Reply:
Now I've pushed that but not the testFlow/ControlJunction re-order.